src/utils.ts

```diff
@@ -20,4 +20,7 @@ export const camelCaseKeys = (rawFlags: LDFlagSet) => {
   return flags;
 };
 
+// TODO: Remove this for the next major version. This was added to maintain backwards compatibility.
```

Comment:
Rather than using a TODO like this, I'd prefer adding a JSDoc with a `@deprecated` tag. Once that's in I can generate the docs locally and make sure it looks acceptable.

[Here](https://github.com/launchdarkly/js-sdk-common/blob/master/typings.d.ts#L216) is some language from an internal dependency which you can copy here.

Reply:
Updated! Sorry for the delay

Reply:
It looks like the `@deprecated` tag is deprecating the `camelCaseKeys` function entirely. Any idea how to just deprecate the `camelCaseKeys. camelCaseKeys` property?

Reply:
I think I figured it out. I had to disable the tslint deprecation rule for that line. I verified that using `camelCaseKeys.camelCaseKeys` now triggers the error, but using `camelCaseKeys` directly does not.